OneCommunityGlobal/HighestGoodNetworkApp
# Peterson_fixed_Checkbox_under_column_Reports_Page_Notification_functi…

# Description
This pull request has been opened to improve the checkbox.

## Related PRS (if any):
None.

## Main changes explained:
The BadgeDevelopment component has been modified to implement the improvement.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4.Log in as an administrator or owner.
5. Go to Badge Management.
6. Click on the checkbox in the column named 'Reports Page Notification'. After clicking, the checkbox should change quickly.

## Screenshots or videos of changes:
![After my fix](https://github.com/user-attachments/assets/0a21509d-ef7a-4199-ba72-6277e9bacd75)

## Note:
None
